packages/react-data-grid/src/Row.tsx

```diff
@@ -3,16 +3,13 @@ import classNames from 'classnames';
 
 import Cell from './Cell';
 import { isFrozen } from './utils/columnUtils';
-import { isPositionStickySupported } from './utils';
 import { IRowRendererProps } from './common/types';
 
 export default class Row<R> extends React.Component<IRowRendererProps<R>> {
   static displayName = 'Row';
 
   static defaultProps = {
-    cellRenderer: Cell,
-    isSelected: false,
-    height: 35
```

Comment:
`isSelected` is not a used prop, and `height` is already default-ed in the root component, so it can't be undefined.